packages/web/src/js/FileUploader.ts

```diff
@@ -91,6 +93,41 @@ class FileUploader extends BaseComponent {
     }
   }
 
+  checkAllowedFileType(file: File) {
+    const fileExtension = file.name.split('.').pop()?.toLowerCase() as string;
+    const fileType = file.type;
+    let isTypeSupported;
+    const shouldValidate = !(this.accept === '' || this.accept === '*' || this.accept === '*/*');
+
+    if (!shouldValidate) {
+      isTypeSupported = true;
+    } else {
+      const acceptArray = this.accept?.replace(/ /g, '').split(',');
+      const acceptExtensions = acceptArray?.filter((exp) => exp.match(/\./g));
+      const acceptTypes = acceptArray?.filter((exp) => exp.match(/\//g));
+
+      acceptExtensions?.forEach((acceptExtension) => {
+        const expression = acceptExtension.replace('.', '').replace('*', '');
+
+        if (fileExtension.match(expression)) {
+          isTypeSupported = true;
+        }
+      });
+
+      acceptTypes?.forEach((acceptType) => {
+        const expression = acceptType.replace('*', '');
+
+        if (fileType.match(expression)) {
+          isTypeSupported = true;
+        }
+      });
+    }
+
+    if (!isTypeSupported) {
+      throw new Error(`"${file.name}" is not supported file`);
```

Comment:
The message could be better, for example:

```suggestion
      throw new Error(`The file "${file.name}" is not supported.`);
```

Or even better, with a suggestion:

```suggestion
      throw new Error(`The file "${file.name}" is not supported. Please ensure you are uploading a supported file format.`);
```

Reply:
whatever, fixed